cli/azd/internal/repository/app_init.go

```diff
@@ -1081,6 +1109,59 @@ func promptSpringBootVersion(console input.Console, ctx context.Context) (string
 	}
 }
 
+func promptBindingDestination(console input.Console, ctx context.Context, bindingName string) (string, error) {
+	for {
+		destination, err := console.Prompt(ctx, input.ConsoleOptions{
+			Message: fmt.Sprintf("Input the value for spring.cloud.stream.bindings.%s.destination", bindingName),
+			Help:    "Hint: Azure Eventhubs Name, please also ensure application properties is well configured.",
+		})
+		if err != nil {
+			return "", err
+		}
+		if IsValidEventhubsName(destination) {
+			return destination, nil
+		} else {
+			console.Message(ctx, "Invalid destination. Please choose another name.")
+		}
+	}
+}
+
+func IsValidEventhubsName(name string) bool {
+	// up to 256 characters
+	if len(name) == 0 || len(name) > 256 {
+		return false
+	}
+	// contain letters, numbers, periods (.), hyphens (-), and underscores (_)
+	// must begin and end with a letter or number
+	regex := regexp.MustCompile(`^[a-zA-Z0-9][a-zA-Z0-9._-]*[a-zA-Z0-9]$`)
+	return regex.MatchString(name)
+}
+
+func promptStorageContainerName(console input.Console, ctx context.Context, key string) (string, error) {
+	for {
+		containerName, err := console.Prompt(ctx, input.ConsoleOptions{
+			Message: fmt.Sprintf("Input the value for %s", key),
```

Comment:
IMU, key will always be this value: `spring.cloud.azure.eventhubs.processor.checkpoint-store.container-name`.

Reply:
As mentioned above, when multi-binders, we can specify different storage container for each binder.